hugo/content/prometheus/_index.md

```diff
@@ -43,7 +43,7 @@ pgMonitor assumes to be using at least Prometheus 2.9.x. We recommend to always
 
 ##### User and Configuration Directory Installation
 
-You will need to create a system user named `ccp_monitoring` which you can do with the following command:
+You will need to create a system user named {{< shell >}}ccp_monitoring{{< /shell >}} which you can do with the following command:
 
 ```bash
 sudo useradd -d /var/lib/ccp_monitoring ccp_monitoring
```

Comment:
if you slap a `-m` on this, you can skip the 'you must create this dir' below

Reply:
Updated this to add the -m. Where are you seeing a reference that says it needs to be created?

Reply:
i was misreading the step below this and not seeing it was a subdir.